ui/app/components/project-init-running.hbs

```diff
@@ -1,5 +1,5 @@
 <p>
-  <Spinner @size="20" />
+  <FlightIcon @name="loading" @size="24" />
```

Comment:
Ditto to https://github.com/hashicorp/waypoint/pull/4147/commits/09dd25ac7059c37372dc5d62833599076305d5a3#r1009967241, I wasn't able to get this state in Mirage (too fast? Tried throttling to 3G in the dev tools), but it looked ok in isolation. Let me know if you have a reliable way to see this state in dev!

Reply:
Here’s how this is currently looking:

<img width="1360" alt="CleanShot 2022-11-01 at 07 11 50@2x" src="https://user-images.githubusercontent.com/34030/199170304-d6e2e6d4-81ef-4ec6-a883-1cc16ec01c83.png">

I used a very kludgey method to simulate this state in Mirage: 53e91a2

Longer-term, we could/should totally add states like this to our Percy suite.

Reply:
And here’s how it looks on `main`:

<img width="1360" alt="CleanShot 2022-11-01 at 07 13 52@2x" src="https://user-images.githubusercontent.com/34030/199170633-4289766c-83ba-4130-b250-1aab18e6db55.png">

Reply:
Thanks, I had trouble getting it to appear, will dig in here

Reply:
Having trouble seeing these in Mirage, so debugged my local setup. 

I ran these steps https://github.com/hashicorp/waypoint/tree/main/ui#running-with-a-local-waypoint-server

At first I got this error:

```
❌ Installing runner...
❌ Initializing Docker client...
! Error installing the runner: Error response from daemon:
Conflict. The container name
  "/waypoint-static-runner" is already in use by container
"b61d5d952f3b46fc29a5d5204c5111c19c5bc2df4fd0bcc8afc250e5e368
You have to
  remove (or rename) that container to be able to reuse that
name.
  
  The Waypoint runner failed to install. This error occurred
after the
  Waypoint server was successfully installed. Your CLI is
configured to
  use the installed server. If you want to retry, you must
uninstall the
  server first.
```

I did a `docker rm` and started over with these same steps https://github.com/hashicorp/waypoint/tree/main/ui#running-with-a-local-waypoint-server. Then I get this error:

```
waypoint (remove-pds-icon) 🌓 ~ waypoint install -platform=docker -docker-server-image="waypoint:dev" -accept-tos

✓ Installing Waypoint server to docker
✓ Installing Waypoint server to docker
✓ Server container started!
✓ Configured server connection
✓ Successfully connected to Waypoint server in Docker!
✓ Server installed and configured!
✓ Runner "static" installed
✓ Waypoint runner installed and started!
❌ Waiting for runner to connect to server at
waypoint-server:9701...
! Error adopting runner: runner not detected by server
after 5 minutes
```

Will xpost in Slack!

Reply:
Curious to hear more about the Mirage issue.

Another option for previewing this state is to fudge the template temporarily so it always displays the project-init-running state:

```diff
  {{page-title @model.name}}

  <PageHeader @iconName="folder">
    <div class="title">
      <h1>{{@model.name}}</h1>
      <small>{{pluralize @model.applicationsList.length "application"}}</small>
    </div>
    <Pds::CtaLink @route="workspace.projects.project.settings" @model={{@model.name}} @variant="ghost "class="pds--iconStart">
      <FlightIcon @name="settings" class="pds-button__iconStart"/>
      &nbsp;Manage settings
    </Pds::CtaLink>
  </PageHeader>

- {{#each @model.applicationsList as |app|}}
-   <!-- ...snip... --> 
- {{/each}}
+ <EmptyState>
+   <ProjectInitRunning @project={{@model}} />
+ </EmptyState>


  {{outlet}}
```